cli/utils/chromedriver-manager.js

```diff
@@ -0,0 +1,69 @@
+/*jslint node: true */
```

Comment:
This is simply refactored from `cli/e2e.js` so that it can be called in `utils/cli-chromedriver-manager-update.js`